pkg/acs/lib/acs2d/do2d.c

```diff
@@ -411,7 +411,7 @@ int Do2D (ACSInfo *acs2d, int extver) {
 	putSingleGroup (acs2d->output, extver, &x, option);
 	if (hstio_err()) {
 		trlerror("Couldn't write imset %d.", extver);
-		return (status = 1001);
+		return (status = INVALID_EXPTIME);
```

Comment:
EXPSCORR indicates each _individual_ exposure from a CR-SPLIT, repeated sub-exposures, or DITHER observation should generate an FLT/FLC file.  In other words the "exps" is misleading in that it does not indicate _exposure time_, but rather individual exposure calibration (to FLT/FLC).  A failure here indicates "WRITE_FAILED".  Or at least this is close to what is happening, but it is still not clear why.